tests/phpunit/compat/class-wc-stripe-subscription-renewal-test.php

```
@@ -782,4 +782,83 @@ public function test_payment_intent_returns_non_retryable_error() {
+		$this->wc_gateway_stripe->process_subscription_payment( $renewal_order->get_total(), $renewal_order, false, false );
+
+		\remove_filter( 'pre_http_request', $pre_http_request_response_callback, 10 );
+
+		$expected_note = sprintf(
+			$expected_note_template,
+			'<a href="' . $request_log_url . '" target="_blank" rel="noopener noreferrer">' . $request_log_url . '</a>'
+		);
+		$note_contents = wp_list_pluck( wc_get_order_notes( [ 'order_id' => $renewal_order->get_id() ] ), 'content' );
+
+		$this->assertContains( $expected_note, $note_contents );
+	}
```

PHPMD (2.15.0)

[error] The method test_failed_renewal_note_links_request_log_url_in_new_tab is not named in camelCase. (undefined)

(CamelCaseMethodName)